plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/DriverTabbedViewer.java

```diff
@@ -349,9 +350,9 @@ public String getToolTipText(Object element) {
                 toolTip.append(driver.getFullName());
                 toolTip.append("\n");
                 if (!usedBy.isEmpty()) {
-                    toolTip.append("Saved connections: ").append(usedBy.size());
+                    toolTip.append(UIConnectionMessages.driver_labal_provider_tip_saved_connections).append(" ").append(usedBy.size());
```

Comment:
Let's embed the count into the localization string to avoid weird concatenations.